javascript/chronometer.js

```diff
@@ -1,26 +1,64 @@
 class Chronometer {
   constructor() {
+    this.currentTime = 0
+    this.intervalId = 0
+  
     // ... your code goes here
   }
   startClick(callback) {
+    
+    let intId = setInterval(()=>{
+      this.currentTime +=1
+    callback()
+  },1000) 
+  
+  
     // ... your code goes here
   }
   getMinutes() {
+    let minutes = 0
+    let time=this.currentTime
+    if (time>=60){
+    minutes=time/60
+    
+    }
+    return Math.floor(minutes)
     // ... your code goes here
   }
+  
   getSeconds() {
+    let seconds=this.currentTime
+    
+    return Math.floor(seconds%60)
+    
     // ... your code goes here
   }
-  twoDigitsNumber() {
+  twoDigitsNumber(time) {
+    if(time<10){
+      return (0+time.toString())
+    }
+    else return (time.toString())
+   
     // ... your code goes here
   }
   stopClick() {
+    clearInterval(this.intId)
```

Comment:
there is no intId attribute, the name is intervalId. by correcting this and the comment on line 10, the stop functionality will now work